plugins/text-freetype2/text-freetype2.c

```diff
@@ -187,6 +187,18 @@ static obs_properties_t *ft2_source_properties(void *unused)
 	obs_properties_add_text(props, "text", obs_module_text("Text"),
 				OBS_TEXT_MULTILINE);
 
+	prop = obs_properties_add_list(props, "alignment",
+				       obs_module_text("Alignment"),
+				       OBS_COMBO_TYPE_LIST,
+				       OBS_COMBO_FORMAT_INT);
+
+	obs_property_list_add_int(prop, obs_module_text("Alignment.Left"),
+				  ALIGN_LEFT);
+	obs_property_list_add_int(prop, obs_module_text("Alignment.Center"),
+				  ALIGN_CENTER);
+	obs_property_list_add_int(prop, obs_module_text("Alignment.Right"),
+				  ALIGN_RIGHT);
+
```

Comment:
Text (GDI+) uses a string type to define the alignment.
As OBS Studio sometimes controlled through obs-websocket, I think the string-type would be better.

https://github.com/obsproject/obs-studio/blob/7778070cbd8e4689d91d90068091ced467c5fdef/plugins/obs-text/gdiplus/obs-text.cpp#L1005-L1008